.gitignore

```diff
@@ -1,5 +1,6 @@
 # Miscellaneous
 *.class
+*.lock
```

Comment:
Need comment out this. Because we need to keep same version of package for each developer